src/js/i18n/locales/es.json

```diff
@@ -125,5 +125,12 @@
   "userFeedback.thankYou": "¡Muchas gracias por ayudarnos a mejorar este sitio web!",
   "userFeedback.characterLimit": "Límite de caracteres",
   "userFeedback.of": "de",
-  "userFeedback.errorMessage": "Lo sentimos mucho, tuvimos problemas para recibir sus comentarios. Por favor inténtelo de nuevo o envíe un correo electrónico a"
+  "userFeedback.errorMessage": "Lo sentimos mucho, tuvimos problemas para recibir sus comentarios. Por favor inténtelo de nuevo o envíe un correo electrónico a",
+  "events.canceled": "Cancelado",
+  "events.free": "Gratis",
+  "events.register": "Regístrese",
+  "events.public": "Abierto al público",
+  "events.details": "Detalles",
+  "events.allEvents": "Todos eventos",
```

Comment:
i will check if this is right, i want to say its "Todos los eventos"

Reply:
just confirmed with my mom & sister, it is "todos los eventos" and not "todos eventos"